### PR TITLE
Add support to save the screenshot in the user's mentioned directory.

### PR DESCRIPTION
**Pull request by akhillb**

### CODE REVIEW

File: lib/commands/actions.js [hunk 1]
```diff
@@ -159,7 +159,8 @@ commands.pullFolder = async function (remotePath) {
 
 commands.getScreenshot = async function () {
   let localFile = temp.path({prefix: 'appium', suffix: '.png'});
-  const png = '/data/local/tmp/screenshot.png';
+  let png_dir = this.opts.androidScreenshotPath || '/data/local/tmp/';
+  const png = png_dir + '/screenshot.png';
```
Comment (imurchie) [inline lib/commands/actions.js]: This should be like `const png = path.resolve(png_dir, 'screenshot.png');`

**Comment by imurchie** [comment]
This desired cap should be added to the constraints, too. https://github.com/appium/appium-android-driver/blob/master/lib/desired-caps.js

**Comment by imurchie** [comment]
This looks good other than a couple of little notes! Thanks for taking this on!

**Comment by akhillb** [comment]
@imurchie thanks for pointing out the things i have missed. Made the changes now.

**Comment by imurchie** [comment]
Very nice! Thank you!

**Comment by imurchie** [comment]
Published in `appium-android-driver@1.10.16`

**Comment by imurchie** [comment]
The final step is to add the capability to the documentation. It should go at the bottom of the "Android only" section of this document: https://github.com/appium/appium/blob/master/docs/en/writing-running-appium/caps.md#android-only

The format should be the capability, a description of what it does (including mentioning the default value), and an example of a possible value.

**Comment by akhillb** [comment]
@imurchie already made that change in this [PR](https://github.com/appium/appium/pull/6649)

**Comment by imurchie** [comment]
Whoops! Thanks!

**Pull request merged**